src/rules/javascript/content-scripts-file-absent.js

```diff
@@ -39,17 +39,26 @@ export default {
             });
             return;
           }
-          let normalizedName = path.resolve(dirname, fileValue);
-          if (path.isAbsolute(fileValue)) {
-            normalizedName = path.join(path.resolve('.'), path.normalize(fileValue));
+
+          // We can't reliably validate relative file names because their
```

Comment:
:+1: Thanks for the additional inline comment, it will be helpful when we will get back to this and we need to recall the reasons why we chose this strategy.

The difference between Firefox and Chrome is not exactly the reason why we chose this strategy, but I guess that is still worth to mention it.

Nit, how about tweaking it a bit? e.g. "... because they are resolved as relative to the current page url on Firefox and the rule itself doesn't know the path of the html file (or files) where the js file is going to be loaded, and so we chose to validate only the absolute file paths to avoid false positive. (Also note that Firefox and Chrome behave differently when resolving relative content script paths used in a tabs.executeScript API call)."

Reply:
> The difference between Firefox and Chrome is not exactly the reason why we chose this strategy, but I guess that is still worth to mention it.

It's not necessarily but that difference helps understanding what developers might prefer and what scenarios might happen more often in the real world.

Anyhow, fixed in 473b1a6